### PR TITLE
[Documentation] useful tools: flops, model converters

### PR DESCRIPTION
**Pull request by zengyh1900**

### CODE REVIEW

File: docs/en/user_guides/useful_tools.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-## Useful tools
+# Useful tools
 
 We provide lots of useful tools under `tools/` directory.
 
-### Get the FLOPs and params (experimental)
+## Get the FLOPs and params
```
Comment (plyfager) [inline docs/en/user_guides/useful_tools.md:5]: I'm not sure whether params equal to parameter amount. Maybe we can introduce what is ``FLOPs`` and ``params`` and why we need these information for starters.

File: docs/en/user_guides/useful_tools.md [hunk 2]
```diff
@@ -32,30 +32,30 @@ Params: 1.52 M
...
 ```shell
-python tools/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}
+python tools/model_converters/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}
 ```
 
 E.g.,
 
 ```shell
-python tools/publish_model.py work_dirs/example_exp/latest.pth example_model_20200202.pth
+python tools/model_converters/publish_model.py work_dirs/example_exp/latest.pth example_model_20200202.pth
```
Comment (plyfager) [inline docs/en/user_guides/useful_tools.md:48]: maybe we should give time tag by script. Example: https://github.com/open-mmlab/mmgeneration/blob/master/tools/publish_model.py#L29.

**Review by plyfager — COMMENTED** [review]
The table in ``List of supported models exportable to ONNX`` has a column note, but seems the  note contain nothing, can we remove this column.

File: docs/en/user_guides/useful_tools.md (hunk 2 above)
Comment (zengyh1900) [reply docs/en/user_guides/useful_tools.md:48]: > maybe we should give time tag by script. Example: https://github.com/open-mmlab/mmgeneration/blob/master/tools/publish_model.py#L29.

to be done in the other PR. 
In this PR, we only discuss docs.

**Pull request merged**